DSL/com.larsreimann.safeds/src/main/kotlin/com/larsreimann/safeds/SafeDS.xtext

```diff
@@ -576,6 +580,19 @@ SdsParenthesizedGoalExpression
     ;
 
 
+/**********************************************************************************************************************
+ * Schemas
+ **********************************************************************************************************************/
+
+SdsColumnList
+: {SdsColumnList} '{' ( columns+=SdsColumn (',' columns+=SdsColumn)* )? '}'
```

Comment:
For the sake of consistency with the other comma-separated lists (besides predicates), please allow a trailing comma here and add an accompanying grammar test.